docs/de/docs/features.md

```diff
@@ -0,0 +1,202 @@
+
+So kann ihr Editor ihnen helfen:
+
+* in <a href="https://code.visualstudio.com/" class="external-link" target="_blank">Visual Studio Code</a>:
+
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+* in <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a>:
+
+![editor support](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+Sie bekommen Autovervollständigung an Stellen, an dennen Sie vorher dies für nicht möglich gehalten haben. Zum Beispiel der `price` Schlüssel aus einem JSON Datensatz (dieser könnte auch verschachtelt sein) aus einer Anfrage.
+
+Hierdurch werden Sie nie wieder einen falschen Schlüsselnamen benutzen und sparen sich lästige Zeit in Dokumentationen, um beispielsweise herauszufinden ob Sie `username` oder `user_name` als Schlüssel verwenden.
+
+### Kompakt
+
+FastAPI nutzt sensible **Standard-Einstellung** für alles, welche aber überall konfiguriert werden können. Alle Parameter können ganz genau an Ihre Bedürfnisse angepasst werden, sodass sie genau die API definieren können, die sie brachen.
+
+Aber standartmäßig, **"funktioniert einfach"** alles.
+
+### Validierung
+
+* Validierung für die meisten (oder alle?) Python **Datentypen**, hierzu gehören:
+    * JSON Objekte (`dict`).
+    * JSON Listen (`list`), die den Typ ihrer Elemente definieren.
+    * Zeichenketten (`str`), mit minimaler und maximaler Länge.
+    * Zahlen (`int`, `float`) mit minimaler und maximaler Größe, etc.
+
+* Validierung für ungewögnliche Typen, wie:
+    * URL.
+    * Email.
+    * UUID.
+    * ... und andere.
+
+Die gesamte Validierung übernimmt das etablierte und robuste **Pydantic**.
+
+### Sicherheit und Authentifizierung
+
+Sicherheit und Authentifizierung integriert. Ohne einen Kompromiss aufgrund einer Datenbank oder den Datenentitäten.
+
+Unterstützt alle Sicherheitsschemata, die von OpenAPI definiert wurden, hierzu gejören:
+
+* HTTP Basis Authentifizierung.
+* **OAuth2** (auch mit **JWT Zugriffstokens**). Schauen Sie hierzu dieses Tutorial [OAuth2 mit JWT](tutorial/security/oauth2-jwt.md){.internal-link target=_blank}.
+* API Schlüssel in:
+    * Kopfzeile (HTTP Header).
+    * Anfrageparametern.
+    * Cookies, etc.
+
+Zusätzlich gibt es alle Sicherheitsfunktionen von Starlette (auch **session cookies**).
+
+Alles wurde als wiederverwendbare Werkzeuge und Komponenten geschaffen, die einfach in ihre Systeme, Datenablagen, relationale und nicht-relationale Datenbanken, ..., integriert werden können.
+
+### Einbringen von Abhängigkeiten (meist: Dependency Injection)
+
+FastAPI bringt ein extrem einfaches, aber extrem mächtiges <abbr title='oft verwendet im Zusammenhang von: Komponenten, Resourcen, Diensten, Dienstanbieter'><strong>Dependency Injection</strong></abbr> System.
+
+* Selbst Abhängigkeiten können Abhängigkeiten haben, woraus eine Hierachie oder ein **"Graph" von Abhängigkeiten** entsteht.
+* **automatische Umsetzung** durch FastAPI.
```

Comment:
This could start with a capital letter too, to be more consistent with the other bullet points.